core/src/main/kotlin/org/evomaster/core/output/naming/LanguageConventionFormatter.kt

```diff
@@ -0,0 +1,35 @@
+package org.evomaster.core.output.naming
+
+import org.evomaster.core.output.OutputFormat
+
+class LanguageConventionFormatter(
+    private val outputFormat: OutputFormat
+) {
+
+    fun formatName(testKeywords: List<String>): String {
+        return when {
+            outputFormat.isJavaOrKotlin() -> formatCamelCase(testKeywords)
+            outputFormat.isPython() -> formatSnakeCase(testKeywords)
+            outputFormat.isJavaScript() -> formatPascalCase(testKeywords)
+            else -> formatCamelCase(testKeywords)
```

Comment:
rather throw an exception on `else`. if one day we add another language, then we want to update all needed code, and this one would be silently ignored

Reply:
Sure, I was actually thinking of C#, but I've just seen it's been commented out from the `OutputFormat` options